modules/mssql/versions.tf

```diff
@@ -20,7 +20,7 @@ terraform {
 
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = ">= 4.22.0, < 5.0"
+      version = ">= 4.28.0, < 5.0"
```

Comment:
Due to this [issue](https://github.com/hashicorp/terraform-provider-google/issues/11891)  we can't go higher than 4.24.0 right now. Let's wait on merging this until this issue is resolved.

Reply:
Just realized this is the beta provider and the issue I mentioned is with GA. We should be good with this change.